aten/src/ATen/native/ao_sparse/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -20,8 +21,6 @@
 
 namespace ao::sparse {
 
-int register_linear_params();
```

Comment:
This isn't avoiding circular dependency linking, right?

Reply:
It is not about circular dependency linking. Now that the header has been included and it works well.